aws/resource_aws_autoscaling_group.go

```diff
@@ -552,6 +552,38 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 					},
 				},
 			},
+
+			"warm_pool": {
+				Type:     schema.TypeSet,
```

Comment:
The linter caught one I missed. When you have a single sub-schema item like this, you should use a `TypeList` instead of a `TypeSet`. Since you don't need the `TypeSet` feature of ignoring order, `TypeList` is more straightforward to use.

Reply:
switched to TypeList